Utilities/DataLoading/EdFi.LoadTools.Test/MappingFactories/ResourceToResourceMetadataMappingFactoryTests.cs

```diff
@@ -11,20 +11,20 @@
 using EdFi.LoadTools.Engine.Factories;
 using EdFi.LoadTools.Engine.Mapping;
 using EdFi.LoadTools.Engine.MappingFactories;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
 
 // ReSharper disable LocalizableElement
 
 namespace EdFi.LoadTools.Test.MappingFactories
 {
-    [TestClass]
+    [TestFixture]
     public class ResourceToResourceMetadataMappingFactoryTests
     {
         private List<JsonModelMetadata> _jsonMetadata;
         private MetadataMapping[] _mappings;
         private List<XmlModelMetadata> _xmlMetadata;
 
-        [TestInitialize]
+        [SetUp]
         public void Initialize()
```

Comment:
In general, it think we should rename this Intialize methods to SetUp in all tests please

Reply:
I did . Thanks